### PR TITLE
modulesync 4.2.0

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 4.2.0

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -225,7 +225,7 @@
   Hash $nginx_upstreams                                   = {},
   Nginx::UpstreamDefaults $nginx_upstreams_defaults       = {},
   Boolean $purge_passenger_repo                           = true,
-  String[1] $nginx_version                                = pick(fact('nginx_version'), '1.6.0'),
+  String[1] $nginx_version                                = pick(fact('nginx_version'), '1.20.0'),
```
Comment (ekohl) [inline manifests/init.pp:228]: This looks like an unrelated change. Is there something so this shows up in the changelog?

Comment (bastelfreak) [reply manifests/init.pp]: removed it
